cluster/gce/gci/configure-helper.sh

```diff
@@ -2578,6 +2578,15 @@ EOF
 function override-kubectl {
     echo "overriding kubectl"
     echo "export PATH=${KUBE_HOME}/bin:\$PATH" > /etc/profile.d/kube_env.sh
+    # Add ${KUBE_HOME}/bin into sudoer secure path.
+    local sudo_path=$(sudo env | grep "^PATH=")
```

Comment:
It's a problem to use `local` and an assignment to command output `=$(...)` in the same statement, because if the command fails then the `set -e` will not make the function fail. This should be in two separate lines.

Reply:
Interesting...I tried, it is as what you said. Thanks for pointing out!

Reply:
Done.